### ISSUE TITLE
Add `sudoers` rules to README to support passwordless provisioning

### ISSUE THREAD

**Issue by gretel** [vagrant-landrush/landrush#190]
Automation should not be prevented by having to enter passwords. So i came up with `sudoers` rules.
on *OS X*. The location of the `sudoers.d` is a bit cumbersome but i guess the commands itself should be the same on many _unices_.

```
sudo visudo -f /private/etc/sudoers.d/vagrant
```

```
Cmnd_Alias VAGRANT_LANDRUSH_HOST_MKDIR = /bin/mkdir /etc/resolver/*
Cmnd_Alias VAGRANT_LANDRUSH_HOST_CP = /bin/cp /*/vagrant_landrush_host_config /etc/resolver/*
Cmnd_Alias VAGRANT_LANDRUSH_HOST_CHMOD = /bin/chmod 644 /etc/resolver/*

%admin ALL=(root) NOPASSWD: VAGRANT_LANDRUSH_HOST_MKDIR, VAGRANT_LANDRUSH_HOST_CP, VAGRANT_LANDRUSH_HOST_CHMOD
```

**Comment by hferentschik** [comment]
So what you are suggesting is to add something to the README, which suggest a manual sudoer config change, so that from then on onwards using Landrush will never ask for the root password again. Sounds reasonable. We might want to add a FAQ section to the README or maybe a Tip&Tricks section. Have you thought about where you want to put this? Maybe you want to create a pull request with the suggested changes?

That said, I am confused about the location of your sudoer config - _/private/etc/sudoers.d/vagrant_. This file does not even exist for me. What's about _/etc/sudoers_? Is that not the file to edit?

**Comment by gretel** [comment]
@hferentschik having sudoers adjusted i something i have done for quite some plugins
> The location of the sudoers.d is a bit cumbersome but i guess the commands itself should be the same on many unices.
i was referring to osx, which i am using.
going to PR if my time allows which i doubt unfortunately. regards

**Comment by hferentschik** [comment]
Resolved by pull request #208
